tests/raftstore/test_snap.rs

```diff
@@ -288,7 +288,7 @@ struct StaleSnap {
 }
 
 impl Filter<RaftMessage> for Arc<StaleSnap> {
-    fn before(&self, msgs: &mut Vec<RaftMessage>) {
+    fn before(&self, msgs: &mut Vec<RaftMessage>) -> FilterResult {
```

Comment:
Maybe `FilterResult` is a bad name. But it's a helpful pattern for a filter to return a flag, and the `SimulateTransport` acts differently according to the returned flag, such as to skip checking whether the messages is dropped, to shortcut the following filters, etc. In the previous comment of the definition of `Filter::before()`
https://github.com/pingcap/tikv/blob/master/tests/raftstore/transport_simulate.rs#L48
(this comment is obsolete since `before` do not return bool any more), this pattern is used by returning a `bool`. A bool is not good for naming and future extension. An enum is better.